Orange/widgets/utils/plot/owplotgui.py

```diff
@@ -572,7 +576,8 @@ def tool_button(self, id, widget):
         '''
         id, name, attr_name, attr_value, callback, icon_name = self._expand_id(id)
         if id == OWPlotGUI.Select:
-            b = self.menu_button(self.Select, [self.SelectionOne, self.SelectionAdd, self.SelectionRemove, self.SelectionToggle], widget)
+            b = self.menu_button(self.Select, [self.SelectionOne, self.SelectionAdd,
+                                               self.SelectionRemove, self.SelectionToggle], widget)
```

Comment:
This call has three arguments. They used to be in a single line that was too long. Now they are in two lines; one and a half in the first line and one and a half in the second. None of this is optimal, but I preferred the original. Or

            b = self.menu_button(
                self.Select,
                [self.SelectionOne, self.SelectionAdd,
                 self.SelectionRemove, self.SelectionToggle],
                widget)

Don't break the lines that are not broken. :)

Reply:
Fixed.